Task-48388  Add link to the perkstore order widget on snapshot page

SaraBoutej opened the pull request:
Task-48388  Add link to the perkstore order widget on snapshot page 

When clicking on perk-store order widget on snapshot page there is nothing happened, So we add link to open My order view when clicking on it

azayati commented:
@SaraBoutej I think that the PR title is not good

rdenarie reviewed (changes requested):
Hello Sara,

Can you update your PR description to follow the commit message best practises : https://developer.exoplatform.org/#id-commits-messages-best-practices

Your description should be on format 
```
<title>
<empty line>
<description answering WHY ? and HOW ? questions>
```

In addition, can put the PR description in the commit message ?

Thanks

SaraBoutej committed:
Task-48388 Add link to the perkstore order widget on snapshot page

When clicking on perk-store order widget on snapshot page there is nothing happened, So we add link to open My order view when clicking on it

SaraBoutej commented:
> Can you update your PR description to follow the commit message best practises : https://developer.exoplatform.org/#id-commits-messages-best-practices

> In addition, can put the PR description in the commit message ?

done @rdenarie

rdenarie reviewed (changes requested):
You update the PR description with a correct message. Can you now update the commit message ?
Thanks

SaraBoutej committed:
Merge branch 'feature/maintenance' into task-48388

SaraBoutej commented:
> You update the PR description with a correct message. Can you now update the commit message ?
> Thanks

done @rdenarie
